.github/workflows/lint-go.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
           test -z "$(git status --porcelain)" || (echo "please run 'go mod tidy && go mod vendor', and submit your changes"; exit 1)
 
   golangci:
-    name: Lint Source Code
+    name: Lint Source Code using golangci
```

Comment:
Two nits, it would be a more consistent name if we change this to `Lint Source Code Golangci`